### ISSUE TITLE
Tighten and even up padding on the homepage

### ISSUE THREAD

**Issue by nedrichards** [flathub/frontend#183]
We need to tighten up and even out the padding and margins on the homepage between the top section, quick setup and the first app section. In particular to balance the needs of the user who doesn't yet know about flathub with the user looking for their first app.
![Screenshot from 2022-01-31 21-04-23](https://user-images.githubusercontent.com/302677/151872924-bd61b189-8b43-4c59-9bdb-dc3711ac3593.png)

**Comment by razzeee** [comment]
I actually would want a graphic for the right side. And probably move the button over to the right too.
Current design is very left heavy.

**Comment by bilelmoussaoui** [comment]
Maybe @jimmac could help with the graphic part? :)

**Comment by nedrichards** [comment]
Yeah, agreed on the left heavy - so long as it all keeps scaling fairly flexibly with different screen sizes a bit more visual pop at the top of the page would be good.

**Comment by razzeee** [comment]
Yeah, we might need to hide a potential graphic on mobile and just show the button in similar fashion to the current design.

I'm doing something similar with the leftside graphic on https://flathub.vercel.app/about in order to keep it readable
